packages/form-layout/src/vaadin-form-item.d.ts

```diff
@@ -43,6 +43,10 @@ import { FormItemMixin } from './vaadin-form-item-mixin.js';
  * because the `label-position` attribute is triggered automatically by the parent
  * `<vaadin-form-layout>`, depending on its width and responsive behavior.
  *
+ * **Deprecation note:** The `label-position` attribute is getting deprecated since
+ * 24.7 and will be removed in Vaddin 25, when a new approach for setting the label
+ * position will be introduced.
```

Comment:
1. Vaddin => Vaadin
2. I guess `getting` could also be removed to keep it simple

```suggestion
 * **Deprecation note:** The `label-position` attribute is deprecated since
 * 24.7 and will be removed in Vaadin 25, when a new approach for setting the label
 * position will be introduced.
```

Reply:
Done.